packages/components/src/MultiSelect/subcomponents/MultiSelectToggle/MultiSelectToggle.tsx

```diff
@@ -38,84 +38,89 @@ export const MultiSelectToggle = forwardRef<
-                  // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
-                  <li key={value} onClick={e => e.stopPropagation()}>
-                    <RemovableTag
-                      removeButtonProps={{
-                        ariaLabel: `Remove option: ${label}`,
-                        onClick: () => onRemoveOption(value),
-                      }}
-                    >
-                      {label}
-                    </RemovableTag>
-                  </li>
-                ))}
-              </ul>
+          <button
+            ref={ref}
+            className={styles.toggleButton}
+            aria-labelledby={ariaLabelledBy}
+            aria-describedby={ariaDescribedBy}
+            aria-controls={ariaControls}
+            aria-expanded={isOpen}
+            aria-haspopup="dialog"
+            type="button"
+            onClick={e => {
+              e.stopPropagation()
+              onClick(e)
+            }}
+          >
+            {isOpen ? (
+              <ChevronUpIcon role="presentation" />
+            ) : (
+              <ChevronDownIcon role="presentation" />
+            )}
+          </button>
 
-              <ClearButton
-                aria-label="Clear all waffles"
-                classNameOverride={styles.clearAllButton}
-                onClick={e => {
-                  e.stopPropagation()
-                  onRemoveAllOptions()
-                }}
-              />
-            </>
-          )}
+          <div
+            className={classnames(
+              styles.selectedItemsContainer,
+              selectedOptions.length && styles.hasSelectedItems
+            )}
+          >
+            {selectedOptions.length > 0 && (
+              <>
+                {/* list-style: none removes role="list" in Safari */}
+                {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+                <ul role="list" className={styles.selectedItems}>
+                  {selectedOptions.map(({ label, value }) => (
+                    // This stops the underlying toggle collapsing the popover when interactive with Tags
+                    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
+                    <li key={value} onClick={e => e.stopPropagation()}>
+                      <RemovableTag
+                        removeButtonProps={{
+                          ariaLabel: `Remove option: ${label}`,
+                          onClick: () => onRemoveOption(value),
+                        }}
+                      >
+                        {label}
+                      </RemovableTag>
+                    </li>
+                  ))}
+                </ul>
+
+                <ClearButton
+                  id={clearAllId}
+                  aria-label="Remove all options from "
+                  aria-labelledby={`${clearAllId} ${ariaLabelledBy}`}
```

Comment:
:O I didn't know this worked. I thought only one of them would be chosen as the label so we had to construct it?

Reply:
Yeah interesting one ay! I noticed react-aria was doing it and it seems to work, though maybe I'll go double check on all of our supported browsers.

The reason why I want to do this instead of creating another visually hidden element is that creating another element would add a nonsensical element to the document for screen reader users.

Reply:
`aria-labelledby` gets priority over `aria-label`, and I guess browsers are smart enough to avoid the recursive issue and reference the accessible name of itself with the `aria-labelledby` ignored.

<img width="1104" alt="image" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/5824caee-fe32-47d9-a11d-eaa140d99b8f">

Reply:
Tested in Chrome, Firefox, Edge on Windows and Chrome, Firefox and Safari on Mac and all working as expected